src/node_context_data.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SRC_NODE_CONTEXT_DATA_H_
+#define SRC_NODE_CONTEXT_DATA_H_
+
+namespace node {
+
+// Pick an index that's hopefully out of the way when we're embedded inside
+// another application. Performance-wise or memory-wise it doesn't matter:
+// Context::SetAlignedPointerInEmbedderData() is backed by a FixedArray,
+// worst case we pay a one-time penalty for resizing the array.
+#ifndef NODE_CONTEXT_EMBEDDER_DATA_INDEX
+#define NODE_CONTEXT_EMBEDDER_DATA_INDEX 32
```

Comment:
Could rename this `EMBEDDER_DATA` to `ENVIRONMENT` as well?

Reply:
i suspect renaming this would make the pr semver major?

Reply:
This macro used to be in `env.h`, which is not a public header.

Reply:
@devsnek raised the argument of Node.js embedding over IRC. I'm fine with removing this macro after a major bump then.